packages/admin/admin/src/EditDialog.test.tsx

```diff
@@ -1,7 +1,7 @@
-import { Button } from "@mui/material";
 import userEvent from "@testing-library/user-event";
 import { render, screen, waitFor } from "test-utils";
 
+import { Button } from "./common/buttons/Button";
```

Comment:
This `Button` needs `variant="textDark"`, right? 
Otherwise it will be the new `primary` variant.